custom_components/tuya_local/devices/ble_smart_plant_moisture.yaml

```diff
@@ -0,0 +1,48 @@
+name: Moisture sensor
+products:
+  - id: bfbf6c7bb11573ed23j4ya
+    model: SGS01
+primary_entity:
+  entity: sensor
+  name: Soil Moisture
+  class: humidity
+  dps:
+      - id: 3
+        name: sensor
+        type: integer
+        force: true
+        unit: "%"
+secondary_entities:
+  - entity: sensor
+    name: Soil Temperature
+    class: temperature
+    dps:
+      - id: 5
+        name: sensor
+        type: integer
+        mapping:
+          - scale: 10
+        unit: C
+  - entity: select
+    name: Temperature unit
```

Comment:
In current head, there should be no name here, instead use `translation_key: temperature_unit`